P5/Source/Specs/am.xml

```diff
@@ -16,10 +16,7 @@ $Id$
   the abbreviation.</desc>
   <desc versionDate="2007-12-20" xml:lang="ko">축약의 확장형에서 생략되거나 대체된 축약형으로, 제시된 문자열 또는 기호열을 포함한다.</desc>
   <desc versionDate="2008-04-06" xml:lang="es">contiene una secuencia de letras o símbolos  presentes en una abreviatura que han sido omitidos o substituidos en la forma extendida de la abreviatura.</desc>
-  <desc versionDate="2008-04-05" xml:lang="ja">
-  当該省略形が，原形を省略またはそれに代わったことを示す文字列または記
-  号列．
-</desc>
+  <desc versionDate="2008-04-05" xml:lang="ja">当該省略形が，原形を省略またはそれに代わったことを示す文字列または記号列．</desc>
```

Comment:
`@versionDate` should be updated

Reply:
Dear Peter, thank you for your cheking. I've updated the date.